docusign/1.0/antora.yml

```diff
@@ -0,0 +1,18 @@
+name: docusign-connector
+version: '1.0'
+display_version: 1.0 (Mule 4)
+title: DocuSign Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables you to interact with the DocuSign Web services API.
+    page-connector-type: Connector
+    page-connector-level: Select
+    page-exchange-group-id: com.mulesoft.connectors
+    page-exchange-asset-id: mule4-docusign-connector
+    page-runtime-version: 4.2.1
+    page-release-notes-page: release-notes::connector/docusign-connector-release-notes-mule-4.adoc
+    page-vendor-name: docusign
+    page-vendor-title: DocuSign
```

Comment:
maybe also here: The certification guys suggested to change the name to DocuSign Connector - Mule 4